core/src/com/unciv/ui/screens/worldscreen/unit/actions/UnitActionModifiers.kt

```diff
@@ -140,7 +159,14 @@ object UnitActionModifiers {
             val statCost = Stats()
             for (conditional in actionUnique.conditionals.filter { it.type == UniqueType.UnitActionStatsCost })
                 statCost.add(conditional.stats)
-            effects += (statCost * -1).toStringOnlyIcons()
+            effects += statCost.toStringOnlyIcons(false)
+        }
+
+        if (actionUnique.conditionals.any { it.type == UniqueType.UnitActionStockpileCost }) {
+            var stockpileString = ""
+            for (conditionals in actionUnique.conditionals.filter { it.type == UniqueType.UnitActionStockpileCost })
+                stockpileString += " ${conditionals.params[0].toInt()} [${conditionals.params[1]}]"
```

Comment:
Will not translate well, replace with curly brackets

Reply:
Check again?